src/openms/source/TRANSFORMATIONS/RAW2PEAK/PeakPickerHiRes.cpp

```diff
@@ -188,11 +188,17 @@ namespace OpenMS
       }
       // MZ spacing sanity checks
       double left_to_central = 0.0, central_to_right = 0.0, min_spacing = 0.0;
+      bool no_left_neighbor = false, no_right_neighbor = false;
```

Comment:
maybe inverse?
bool no_left_neighbor -> has_left_neighbor
could remove the double negations !no_...

Reply:
Sounds good to me! I will revise these when I make an updated commit :)